src/app/components/ItemFilters/ItemFilter.jsx

```diff
@@ -92,29 +94,33 @@ const ItemFilter = ({
   const clear = () => {
     const clear = true;
     setSelectionMade(true);
-    setSelectedFilters(prevSelectedFilters => ({
-      ...prevSelectedFilters,
-      [filter]: [],
+    setSelectedFields(prevSelectedFields => ({
+      ...prevSelectedFields,
+      [field]: [],
     }));
     submitFilterSelections && submitFilterSelections(clear);
   };
 
-  const isSelected = (option) =>
-    isOptionSelected(selectedFilters[filter], option.id);
+  const isSelected = (option) => {
+    return isOptionSelected(selectedFields[field], option.value);
+  }
   const updatedOptions = updateOptions(options);
   const determineNumOfSelections = () => {
-    const thisFilterSelections = initialFilters[filter];
-    const numSelection = thisFilterSelections.length === 0 ? '' :
-      typeof thisFilterSelections === 'string' ? 1 : thisFilterSelections.length;
+    const thisFilterSelections = initialFilters[field];
+    let numSelection
+    if (thisFilterSelections) {
+      const labels = [...new Set(getLabelsForValues(thisFilterSelections, field, fieldToOptionsMap))]
```

Comment:
Might be useful comment here to say we are deduping because of offsite

Reply:
on closer inspection, i think this was an over correction so im removing the Set instantiation